distributed/utils_test.py

```diff
@@ -94,6 +95,17 @@ def valid_python_script(tmpdir_factory):
     return local_file
 
 
+@pytest.fixture(scope="session", autouse=True)
+def event_loop_policy():
+    try:
+        # FIXME: jupyter_client 7.0-7.0.1 clobbers asyncio policy at import
+        # so make sure to import it first before setting our policy
+        import jupyter_client  # noqa: F401
+    except ImportError:
+        pass
+    asyncio.set_event_loop_policy(AnyThreadEventLoopPolicy())
+
+
```

Comment:
Thanks for your work on this @minrk

>  jupyter-client 7.0.2 will probably be out later today, so I'd just wait for that, since it'll fix the issue and obviate the need for event loop policy workarounds.

That sounds good to me 👍 It looks like 7.0.2 is out https://pypi.org/project/jupyter-client/7.0.2/